scripts/client_test/cli_test.py

```diff
@@ -255,18 +255,17 @@ def test_all(self) -> None:
         for name, rt in RUNTIMES.items():
             self.build_runtime(rt["workdir"])
 
-        processes = [
-            subprocess.Popen(
+        for name, expl in EXAMPLES.items():
+            p = subprocess.Popen(
```

Comment:
add cwd param to function of invoke and use it?

Reply:
the logging seams not working propertlly in function of invoke, let's do it in another pr